src/tests/test_networkgroup.py

```diff
@@ -2,42 +2,35 @@
 Test for network group
 """
 import unittest
-
-from .common import skip_python3
-
-skip_python3()
+from .common import cleanup
 
 
 class TestNetworkGroup(unittest.TestCase):
     """
     Test case for network group
     """
+
+    def tearDown(self):
+        cleanup()
```

Comment:
I don't think you need this here. Can you show a corresponding failing check?

Reply:
I was getting this error ```'Found application file keys.dat in src dir'```

So, I added tearDown() that fixed it

Reply:
I will retest removing it, once more

Reply:
please post the link

Reply:
https://github.com/Bitmessage/PyBitmessage/pull/1792/checks?sha=8d545fa066cdda420e1450ca248dc79ee1f74fba

I can't find the link for that logs, but it was for this commit

Reply:
Probably it's because you should've remove this module, not add another bikeshed, see. below.